lib/cocoapods/command/deintegrate.rb

```diff
@@ -1,5 +1,7 @@
 module Pod
   class Command
+    # @since 1.0.0.beta.1
```

Comment:
Well technically `pod deintegrate` has existed in this gem since its first release